sky/adaptors/azure.py

```diff
@@ -46,6 +46,13 @@ def http_error_exception():
     return exceptions.HttpResponseError
 
 
+@import_package
+def resources_not_found_exception():
+    """ResourceNotFoundError exception."""
+    from azure.core import exceptions
+    return exceptions.ResourceNotFoundError
```

Comment:
Should we combine this and the `http_error_exception()` above into
```
@import_package
def exceptions():
    """ResourceNotFoundError exception."""
    from azure.core import exceptions
    return exceptions
```

and in all the callers, we use `azure.exceptions().specific_exception`?

Reply:
SG! Changed.